ixmp4/cli/platforms.py

```diff
@@ -97,8 +97,7 @@ def prompt_sqlite_removal(dsn: str):
     path = Path(dsn.replace("sqlite://", ""))
     path_str = typer.style(path, fg=typer.colors.CYAN)
     if typer.confirm(
-        "Do you want to remove the associated database file at "
-        f"{path_str} aswell?"  # type: ignore
+        "Do you want to remove the associated database file at " f"{path_str} aswell?"  # type: ignore
```

Comment:
"aswell" should be written "as well", but "also" is even shorter, I guess...

```suggestion
        "Do you want to also remove the associated database file at " f"{path_str}?"  # type: ignore
```